lib/components/Atom/WheelPicker/wheel-picker.scss

```diff
@@ -0,0 +1,66 @@
+@import "@deriv-com/quill-tokens/dist/breakpoints.scss";
+
+.quill-wheel-picker {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    height: 100%;
+    background-color: var(--component-field-bg-outline-default);
+    padding: 0;
+    @include breakpoint("md") {
+        padding: var(--component-actionSheet-spacing-padding-sm);
+        border-radius: var(--core-borderRadius-400);
+        box-shadow: var(--semantic-elevation-shadow-card-hover);
+    }
+
+    &__container {
+        position: relative;
+        margin: 0;
+        height: 100%;
+        width: 100%;
+        scrollbar-width: none;
+        @include breakpoint("md") {
+            height: 240px;
+        }
+    }
+
+    &__data-items {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        overflow-y: scroll;
+        scroll-snap-type: y mandatory;
+        padding: 0;
+        -ms-overflow-style: none;
+        scrollbar-width: none;
```

Comment:
Please, check if will work in Real IOS (Safari). In my previous PR I found out, that `scrollbar-width` css property is not working there, so I added 
```
            &::-webkit-scrollbar {
                display: none; /* Safari */
            }
![Uploading Screenshot 2024-07-30 at 2.44.16 PM.png…]()

```

Reply:
![Screenshot 2024-07-30 at 2 44 16 PM](https://github.com/user-attachments/assets/d7904232-2d7e-44ff-94a5-e5326a339bc8)

Reply:
Oh, sorry, found it below, ignore my comment)))))